function-app/adb-to-purview/src/Function.Domain/Helpers/OlProcessing/ValidateOlEvent.cs

```diff
@@ -75,28 +75,42 @@ private bool ValidateSynapseOlEvent(Event olEvent)
             // The data sources are never different ones where consolidation would resolve it. 
             // The ones that have both show the same datasource in  I/O.
             // Currently the assumption is that the first input is the sink and the rest of the inputs are the source(s).
-            if (olEvent.Inputs.Count == 0)
-            {
-                return false;
-            }
-            if (olEvent.EventType != "COMPLETE")// && olEvent.EventType != "START")
-            {
-                return false;
-            }
-            if (olEvent.Inputs.Count <= 1 && olEvent.Outputs.Count == 0)
-            {
-                return false;
-            }
-            if (olEvent.Inputs.Count == 0 && olEvent.Outputs.Count > 0)
-            {
-                return false;
-            }
-            if (InOutEqual(olEvent))
+            if (olEvent.Inputs.Count == 0 && olEvent.Outputs.Count == 0)
             {
                 return false;
             }
+            //if (InOutEqual(olEvent))   // TO DO check mani  and the above comments
```

Comment:
Please remove this commented code. We agreed that the logic is not needed in our case.

Reply:
Done